python/README.adoc

```diff
@@ -1,8 +1,30 @@
 = README
+pre-install: 
+brew install librdkafka
+brew cask install homebrew/cask-versions/adoptopenjdk8
+brew install kafka
```

Comment:
Do we need the kafka setup, given that the docker-compose file contains a Kafka installation?
I'd prefer to use that one and not have that in the "base os". 
This also is more machine-agnostic, while `brew` is pretty much OSX only.

Reply:
Right, I will use the docker-compose instead, it was recording what I was done locally 👍